stock_move_line_lock_qty_done/models/res_config_settings.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2024 Quartile
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    lock_qty_done = fields.Boolean(
```

Comment:
I'd prefer a related on a company field.

That will allow you to avoid `get/set_values` calls.

Reply:
If we related on a company field instead of system parameter and enable this feature by default, do you want to set `lock_qty_done` to `false` at the end of the test cases of the module? May be I don't get your intention.

Reply:
No.

From a functional point of view, enabling or disabling the feature per company is a good practice.
From a technical point of view (that's a matter of taste), I don't like such writings. Moreover, in v16, you have the field attribute `config_parameter`that allows to avoid such `get/set`.